content/client-lib-development-guide/features.textile

```diff
@@ -262,6 +263,7 @@ h3(#realtimeclient). RealtimeClient
 * @(RTC6)@ @RealtimeClient#time@ function:
 ** @(RTC6a)@ Proxy to @RestClient#time@ presented with an async or threaded interface as appropriate
 * @(RTC7)@ The client library must use the configured timeouts specified in the @ClientOptions@, falling back to the "client library defaults":#defaults and defaults described in @ClientOptions@ below
+* @(RTC8)@ Authentication upgrades are only possible when the client is @CONNECTED@ or @CONNECTING@. If @auth.authorise@ is called with @AuthOptions#force@ set to @true@ and the state is @CONNECTED@ or @CONNECTING@, the client will obtain a new token, disconnect the current transport and resume the connection (see "@RTN15c@":#RTN15c for details on resume). A test should exist that performs an upgrade of capabilities without any message loss during the upgrade process. A test should also for an auth failure which should result in the connection entering the @FAILED@ state
```

Comment:
Let's not use "upgrade" in this context because of the conflict with transport upgrade. We could refer to token changes, updates, or renewals.

If we support this operation in the `CONNECTING` state, why not also when `DISCONNECTED` ?